src/Core/functions.php

```diff
@@ -28,7 +28,7 @@
      * @param string|array|object $text Text to wrap through htmlspecialchars. Also works with arrays, and objects.
      *    Arrays will be mapped and have all their elements escaped. Objects will be string cast if they
      *    implement a `__toString` method. Otherwise the class name will be used.
-     * @param bool $double Encode existing html entities.
+     * @param bool|string $double Encode existing html entities. If string it will be used as character set.
```

Comment:
Should we remove those shim behaviors for a cleaner api? Or do we go with $options array here?